### PR TITLE
fix lint errors

### PR DESCRIPTION
**Pull request by ariquinones**

### CODE REVIEW

File: src/sdk/service/upload-service/upload-service.ts [hunk 1]
```diff
@@ -188,6 +188,7 @@ export abstract class UploadService {
           });
         });
       }
+      /* tslint:disable-next-line */
```
Comment (cfsnyder) [inline src/sdk/service/upload-service/upload-service.ts]: what did you need to disable on this line @ariquinones ?

Comment (ariquinones) [reply src/sdk/service/upload-service/upload-service.ts]: it was a lint error as well saying you don't use that variable

Comment (ariquinones) [reply src/sdk/service/upload-service/upload-service.ts]: const client = this.getHTTPClient();

Comment (cfsnyder) [reply src/sdk/service/upload-service/upload-service.ts]: ok, we just need to remove the line vs. ignore it then. (and thanks for fixing my lint errors)

Comment (ariquinones) [reply src/sdk/service/upload-service/upload-service.ts]: Okay updated

**Comment by cfsnyder** [comment]
Thanks @ariquinones !

**Pull request merged**